life_expectancy/cleaning.py

```diff
@@ -23,91 +22,132 @@
 COMPOSED_COL = 'unit,sex,age,geo\\time'
 DECOMPOSED_COLs = ['unit', 'sex', 'age','region']
 
-def clean_data(region_filter: str) -> None:
+def load_data (input_file_path: Path = INPUT_FILE_PATH) -> pd.DataFrame:
```

Comment:
não sabia que Path era uma classe. Talvez obrigar o pessoal a usar o path não seja assim tão bom, mas at this point deve ser bastante comum

Reply:
sim, usei o módulo pathlib e para ser correto o type é esse.
usei a pathlib pois é mais portável vs a "import os", é indep de os, creio que é tb mais legível.

Reply:
Não tenha medo das classes, André! Elas são amigas!